File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    let count = 0
    let l = 0
    const arr = [...s]
    
    arr.forEach((value) => {
        l++
        let i = value.charCodeAt();
        if((i >= 48) && (i <= 57)) {
            count++
        }
    })
    
    if(count === s.length) {
        if((l === 4) || (l === 6)) {
            return true   
        }
    }
    return false
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

bool solution(string s) {
    
    if(!(s.size() == 4 || s.size() == 6)) return false;
    
    for (int i=0; i<s.size(); i++) {
        //if (s[i] < '0' || s[i] > '9') return false;
        if(!isdigit(s[i])) return false;
    }
    
    return true;
}